docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: "3"
+
+services: 
+     
+  db:
+    image: mariadb:latest
+    container_name: mariadb-server
+    restart: always
+    command: --default-authentication-plugin=mysql_native_password
+    environment:
+      MYSQL_ROOT_PASSWORD: reidb  
```

Comment:
Never push passwords to github. Use environment variables to populate the compose file at runtime. It is fine for this lesson, just keep it in mind for the next.